core/frontend/src/EmphasizeElements.ts

```diff
@@ -234,15 +234,34 @@ export class EmphasizeElements implements FeatureOverrideProvider {
     return true;
   }
 
-  /** Clear elements with color/transparency overrides. Specify key to clear only a single override.
+  /** Clear color/transparency overrides from elements. Removes all overrides when neither key or ids is specified.
+   * @params key Specify key value from [[EmphasizeElements.getOverriddenElements]] or [[EmphasizeElements.createOverrideKey]]
+   * to remove a single color/transparency override for the corresponding elements.
+   * @params ids Specify the IDs of elements to remove any color/transparency override from.
    * @return false if nothing to clear.
    */
-  public clearOverriddenElements(vp: Viewport, key?: number): boolean {
+  public clearOverriddenElements(vp: Viewport, key?: number, ids?: Id64Arg): boolean {

```

Comment:
Maybe a nitpick but what does this do if you give it both `keys` and `ids`? Shouldn't it take `number | Id64Arg` instead of the two args?

Reply:
Just does key, number | Id64rg will be better.